modules/administration/pages/ptfs.adoc

```diff
@@ -0,0 +1,81 @@
+[[ptfs]]
+= Using PTFs in {productname}
+
+SUSE provide for their Enterprise Products temporary fixes shipped to individual customers.
+These PTFs (Program Temporary Fixes) can now be made available as repositories which can be synced in {productname}.
+
+
+== Understanding PTF packages
+
+PTF packages are installed via a proxy package and are named `ptf-xxxxxx`.
+They will depend on the correct version of the package that is known to include the correction in the software. This type of package
+
+* cannot be installed accidentally (ie. zypper update will never suggest installing them),
+* cannot be removed accidentally (ie. a newer package version will not replace the PTF one, unless the user makes it explicitly on the zypper command line),
+* is only updated when the newer version is known to address the specific issue addressed first by the PTF,
+* updates only packages already installed on the system (a software that is split in multiple packages will have only the packages currently installed on the system being replaced).
+
+The correct ID of the package will be provided by SUSE Support during the course of the support case investigation, along with instruction on how to deploy/restart the affected services.
+
+
+== Installing PTF packages
+
+
+[NOTE]
+====
+PTF packages are currently only supported for SLE 12 and SLE 15 based systems.
+Other versions or Operating Systems do not have this feature yet and the pages are not visible for them.
+====
+
+
+.Procedure: Enable and Synchronize PTF repositories using the command line
+
+. On the console enter ``mgr-sync refresh``
+. Enter ``mgr-sync list channel`` and look for channels starting with your SCC account name and `ptfs` in its name. E.g. `a123456-sles-15.3-ptfs-x86_64`
+. Enable the PTF channel with ``mgr-sync add channel <label>``
+
+Now this channel is available and can be added to every system which is using the same base channel.
+
+PTF packages needs to be installed explicit. They are not automatically installed when updating a system.
+The SUSE Customer Support will tell the ptf number to fix a specific problem.
+In {productname} {webui} every system with PTFs available for installation has a page which list them.
```

Comment:
```suggestion
PTF packages need to be installed explicitly, since they are not automatically picked up when updating a system.
The SUSE Customer Support will provide the PTF number to fix a specific problem.
In {productname} {webui} every system with PTFs available for installation has a page which list them.
```

Reply:
used it